2015/proposals/agarwal-rajat.md

```diff
@@ -0,0 +1,92 @@
+# Title
+
+Add taxonomic name resolution to the EcoData Retriever to facilitate data science approaches to ecology
+
+## Abstract
+
+The EcoData Retriever is a Python based tool for automatically downloading, cleaning up, and restructuring ecological data. It does the data munging so that the users can focus on using the data.
+As in ecological and evolutionary data, the names of species are constantly being redefined - it is difficult to combine datasets. By automating reconciliation of different species names as part of the process of accessing the data in the first place it will become much easier to combine diverse datasets and in new and interesting ways.
```

Comment:
"As in" -> "In"

" - it is" -> "This makes it"

"become much easier" -> "be easier"